pkg/monitoring/metrics/common/client/rt_wrapper.go

```diff
@@ -41,58 +46,33 @@ func (r *rtWrapper) RoundTrip(request *http.Request) (response *http.Response, e
 		status = strconv.Itoa(response.StatusCode)
 	}
 
-	host = "none"
-	if request.URL != nil {
-		host = request.URL.Host
-	}
-
 	resource, verb = parseURLResourceOperation(request)
-	if verb == "" {
-		verb = "none"
-	}
-	if resource == "" {
-		resource = "none"
-	}
+	host = getHost(request)
+
 	requestResult.WithLabelValues(status, request.Method, host, resource, verb).Add(1)
 
 	return response, err
 }
 
-func parseURLResourceOperation(request *http.Request) (resource string, verb string) {
+func parseURLResourceOperation(request *http.Request) (resource, verb string) {
 	method := request.Method
-
-	resource = ""
-	verb = ""
-
-	if request.URL.Path == "" || method == "" {
-		return
-	}
-
-	for _, r := range resourceParsingRegexs {
-		if resource != "" {
-			break
-		}
-		match := r.FindStringSubmatch(request.URL.Path)
-		if len(match) > 1 {
-			resource = match[1]
-		}
+	if request.URL == nil || method == "" {
+		return defaultNone, defaultNone
 	}
 
+	resource = findResource(*request.URL)
 	if resource == "" {
-		return
+		return defaultNone, defaultNone
 	}
 
-	switch method {
+	verb = determineVerb(*request.URL, method)
+	return resource, verb
+}
+
+func determineVerb(u url.URL, methodOrVerb string) (verb string) {
```

Comment:
Can you make the funcs name more descriptive? Something like `getVerbFromHTTPVerb`?